plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergMaterializedViewTest.java

```diff
@@ -87,6 +87,18 @@ public void testShowTables()
         assertUpdate("DROP MATERIALIZED VIEW materialized_view_show_tables_test");
     }
 
+    @Test
+    public void testCommentColumnMaterializedView()
+    {
+        String viewColumnName = "_bigint";
+        String materializedViewName = "test_materialized_view_" + randomNameSuffix();
+        assertUpdate(format("CREATE MATERIALIZED VIEW %s AS SELECT * FROM base_table1", materializedViewName));
+        assertUpdate(format("COMMENT ON COLUMN %s.%s IS 'new region key comment'", materializedViewName, viewColumnName));
+        assertThat(getColumnComment(materializedViewName, viewColumnName)).isEqualTo("new region key comment");
```

Comment:
The comment `new region key comment` is weird to me because the column name is `_bigint`.

Reply:
refactored